### DISCUSSION TITLE
Is it possible to use psiCollectCron with puppeterScript?

### DISCUSSION THREAD

**Discussion by poteirard** [GoogleChrome/lighthouse-ci#693] [Q&A] [answered]
Hello,

First of all thanks for this amazing project. It's amazing how useful it is.

I was wondering if will it be possible to use psiCollectCron with puppeterScript in the server. What I'm trying to find is a way to run the lighthouse periodically (cron) but also to pass the auth flow.

Is it possible? I tried to follow the code but I got a bit lost 😅

**Comment by patrickhulce** [comment, marked as answer]
Thanks @poteirard! :)

> I was wondering if will it be possible to use psiCollectCron with puppeterScript in the server.

It is not. PSI collection happens on Google servers somewhere while puppeteerScript is part of the local `lhci collect` that runs on the local machine. If you want to locally run collection on a schedule, you'll have to setup a regular old cron job that invokes `lhci collect` instead.
